src/components/confirm/confirm.stories.mdx

```diff
@@ -74,6 +74,171 @@ const MyComponent = ({ isOpen, handleClose, handleConfirm }) => (
   </Story>
 </Preview>
 
+
+### Single action
+
+<Preview>
+  <Story name="single action" parameters={{ chromatic: { disable: true }}}>
+    {() => {
+      const [isOpen, setIsOpen] = useState(false);
+      return (
+        <>
+          <Button onClick={ () => setIsOpen(!isOpen) }>
+            Open Confirm
+          </Button>
+          <Confirm
+            title="Are you sure?"
+            subtitle="Subtitle"
+            open={ isOpen }
+            onConfirm={ () => setIsOpen(false) }
+          >
+            Content
+          </Confirm>
+        </>
+      )
+    }}
+  </Story>
+</Preview>
+
+
+## `destructive`
+
+<Preview>
+  <Story name="confirm - destructive" parameters={{ chromatic: { disable: true }}}>
+    {() => {
+      const [isOpen, setIsOpen] = useState(false);
+      return (
+        <>
+          <Button onClick={ () => setIsOpen(!isOpen) }>
+            Open Confirm
+          </Button>
+          <Confirm
+            title="Are you sure?"
+            subtitle="Subtitle"
+            destructive
+            cancelButtonType="tertiary"
+            open={ isOpen }
+            onConfirm={ () => setIsOpen(false) }
+            onCancel={ () => setIsOpen(false) }
+          >
+            Content
+          </Confirm>
+        </>
+      )
+    }}
+  </Story>
+</Preview>
+
+## `disableConfirm`
```

Comment:
These titles should be descriptive, not just the name of the prop. So this one would be `Disable confirm button` or similar